src/App.js

```diff
@@ -2,12 +2,71 @@ import React from "react";
 import "./App.css";
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      current: 0,
+      plus: 1,
+    };
+  }
+
+  addHandle = () => {
+    this.setState({
+      current: this.state.current + this.state.plus,
+    });
+  };
```

Comment:
```suggestion
  addHandle = () => {
    this.setState((prevState) => {
    return {current: prevState.count + prevState.plus}
  });
  };
```
The way you have it already is fine in this case and in line with what we've learned so far. The suggestion I made here is more advanced and a less intuitive for beginners, so we haven't touched on it yet in class. I wanted to show it to you here because depending on the logic in your application, it could be necessary to update state this way and I figured you would like to see that this is an option. 

setState can accept a callback with a parameter that represents the previous state. Here's the documentation on when and why to update state this way: https://reactjs.org/docs/faq-state.html
and here's some more info on it: https://dev.to/nicolelc16/react-previous-state-4akk#:~:text=state%20in%20setState()%20.,on%20what%20it%20was%20before.